pkg/collector/python/init.go

```diff
@@ -422,6 +469,14 @@ func Initialize(paths ...string) error {
 		}
 	}
 
+	// If we want to report the packages version
+	if pkgconfigsetup.Datadog().GetBool("telemetry.python_packages") {
+		// Get the python packages version
+		// New packages can be installed, but they're not taken into account until the agent is restarted,
+		// so it's safe to cache the versions here.
+		cache.Cache.Set(pythonPackagesCacheKey, getPythonPackagesVersion(pythonBinPath), cache.NoExpiration)
```

Comment:
What if someone start the agent, install a new package and upgrade the Agent ? Unless I'm missing something we would miss the newly installed package, right ?

Is that an issue ?

Reply:
If they upgrade the agent there will be a "restart", so we should re-run this part